src/toM3u8.js

```diff
@@ -141,7 +141,8 @@ export const toM3u8 = dashPlaylists => {
     master.mediaGroups.AUDIO.audio = organizeAudioPlaylists(audioPlaylists);
   }
 
-  if (vttPlaylists.length) {
+  // turn off vtt playlist support until issues with VHS are resolved
```

Comment:
I'm not sure if we should directly reference VHS here. If it's an issue with VHS, then we can make the change in that repo. mpd-parser should be "independent" when it can be. If it's an issue with how we're handling the parsing in mpd-parser then we can turn it off here and reference that.

Reply:
Also, if it is an issue here, we should add a `TODO`

Reply:
I believe this is an issue here and at the moment this parser produces an output that only really makes sense with VHS. I'll add the TODO

Reply:
I do agree that right now it is primarily geared towards VHS, however, it could be used for other reasons (just as the m3u8-parser can). Either way though, since the issue may be here, it may be best to simple say `TODO: vtt playlists not yet supported`

Reply:
Updated the note